gtests/net/mptcp/syscalls/connect_send.pkt

```diff
@@ -0,0 +1,24 @@
+// connect() function, connection initiated by the kernel
+// ensure we get send() wake-up events correctly
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
+0.0   socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0  fcntl(3, F_GETFL) = 0x2 (flags O_RDWR)
++0.0  fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
+// Establish connection and verify blocking write will be woken-up by
+// MPC completion
+
++0.0  connect(3, ..., ...) = -1 EINPROGRESS (Operation now in progress)
++0.0   > S 0:0(0) <mss 1460,sackOK,TS val 100 ecr 0,nop,wscale 8,mpcapable v1 flags[flag_h] nokey>
+
+
++0.0  send(3, ..., 400, 0) = -1  EAGAIN (Resource temporarily unavailable)
++0.0  fcntl(3, F_SETFL, O_RDWR) = 0
+
++0.02  < S. 0:0(0) ack 1 win 65535 <mss 1460,sackOK,TS val 700 ecr 100,nop,wscale 8,mpcapable v1 flags[flag_h] key[skey=2] >
++0.0   > . 1:1(0) ack 1 <nop,nop,TS val 100 ecr 700,mpcapable v1 flags[flag_h] key[ckey,skey] >
+
++0...0.020 send(3, ..., 400, 0) = 400
```

Comment:
I guess we don't need to check here that packets have been properly sent (MPC completion with data), right?

Reply:
yes, that should be covered by DSS-specific pktdrills